globals.d.ts

```diff
@@ -59,6 +59,7 @@ declare module 'react-native-dotenv' {
   export const CLOUDINARY_API_KEY: string;
   export const CLOUDINARY_API_SECRET: string;
   export const CLOUDINARY_CLOUD_NAME: string;
+  export const NOTIFICATIONS_API_KEY: string;
```

Comment:
This was merged in rainbow env here: https://github.com/rainbow-me/rainbow-env/pull/52